src/main/java/org/takes/servlet/HttpServletResponseFake.java

```diff
@@ -118,13 +123,16 @@ public void sendError(
     @Override
     public Collection<String> getHeaders(final String header) {
         final String prefix = String.format(
-            "%s:", new EnglishLowerCase(header)
-        );
+            "%s:",
+            new UncheckedText(
+                new Lowered(header)
+            ));
         try {
             return new ListOf<>(
                 new Filtered<>(
-                    hdr -> !new EnglishLowerCase(hdr).string()
-                        .startsWith(prefix),
+                    hdr -> !new UncheckedText(
+                        new Lowered(hdr)
+                    ).asString().startsWith(prefix),
```

Comment:
@fanifieiev please, leave a puzzle for [it](https://github.com/yegor256/cactoos/issues/1265).